docs/sources/configuration/integrations/cloudwatch-exporter-config.md

```diff
@@ -0,0 +1,400 @@
+      "Action": [
+        "tag:GetResources",
+        "cloudwatch:GetMetricData",
+        "cloudwatch:GetMetricStatistics",
+        "cloudwatch:ListMetrics",
+        "ec2:DescribeTags",
+        "ec2:DescribeInstances",
+        "ec2:DescribeRegions",
+        "ec2:DescribeTransitGateway*",
+        "apigateway:GET",
+        "dms:DescribeReplicationInstances",
+        "dms:DescribeReplicationTasks"
+      ],
+      "Effect": "Allow",
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+## Configuration options
+
+Configuration reference:
+
+```yaml
+  #
+  # Common Integration Settings
+  #
+    
+  # Enables the cloudwatch_exporter integration, allowing the Agent to automatically
+  # collect CloudWatch metrics as configured.
+  [enabled: <boolean> | default = false]
+
+  # Sets an explicit value for the instance label when the integration is
+  # self-scraped. Overrides inferred values.
+  #
+  # The default value for this integration is a hash of the whole integration configuration.
+  [instance: <string>]
+
+  # Automatically collect metrics from this integration. If disabled,
+  # the cloudwatch_exporter integration is run but not scraped and thus not
+  # remote-written. Metrics for the integration are exposed at
+  # /integrations/cloudwatch_exporter/metrics and can be scraped by an external
+  # process.
+  [scrape_integration: <boolean> | default = <integrations_config.scrape_integrations>]
+
+  # How often should the metrics be collected. Defaults to
+  # prometheus.global.scrape_interval.
+  [scrape_interval: <duration> | default = <global_config.scrape_interval>]
+
+  # The timeout before considering the scrape a failure. Defaults to
+  # prometheus.global.scrape_timeout.
+  [scrape_timeout: <duration> | default = <global_config.scrape_timeout>]
+
+  # Allows for relabeling labels on the target.
+  relabel_configs:
+    [- <relabel_config> ... ]
+
+  # Relabel metrics coming from the integration, allowing to drop series
+  # from the integration that you don't care about.
+  metric_relabel_configs:
+    [ - <relabel_config> ... ]
+
+  # How frequent to truncate the WAL for this integration.
+  [wal_truncate_frequency: <duration> | default = "60m"]
+
+  #
+  # Exporter-specific configuration options
+  #
+
+  # Required: AWS region to use when calling STS (https://docs.aws.amazon.com/STS/latest/APIReference/welcome.html) for retrieving
+  # account information.
+  # Ex: us-east-2
+  sts_region: <string>
+
+  discovery:
+
+    # Optional: List of tags (value) per service (key) to export in all metrics. For example defining the ["name", "type"] under
+    # AWS/EC2 will export the name and type tags and its values as labels in all metrics. Affects all discovery jobs.
+    # Ex:
+    # exported_tags:
+    #   AWS/EC2:
+    #     - name
+    exported_tags:
+      { <string>: [ <string> ] }
+
+    # List of discovery jobs
+    jobs: [ <discovery_job> ]
+
+  # List of static jobs
+  static: [ <static_job> ]
+```
+
+# discovery_job
```

Comment:
Are all these sections supposed to be at this heading level (`#`)?

Reply:
Nice catch! Fixing